astroplan/tests/test_core.py

```diff
@@ -155,3 +152,591 @@ def test_FixedTarget_ra_dec():
                                                         'SkyCoord')
     assert vega.coord.dec == vega_coords.dec == vega.dec, ('Retrieve Dec from '
                                                            'SkyCoord')
+
+def test_sunrise_sunset_equator():
+    '''
+    Check that time of sunrise/set for an observer on the equator is
+    consistent with PyEphem results (for no atmosphere/pressure=0)
```

Comment:
[USNO uses some assumptions](http://aa.usno.navy.mil/faq/docs/RST_defs.php#riseset) about the atmosphere and which limb of the sun/moon defines the rise/set, which might make it difficult to compare without knowing their assumptions better. Do you know where to look for that?